library/qhull-2012.1/Announce.txt

```diff
@@ -38,8 +38,7 @@ The convex hull of a set of points is the smallest convex set that contains
 the points.  This article presents a practical convex hull algorithm that
 combines the two-dimensional Quickhull Algorithm with  the general dimension
 Beneath-Beyond Algorithm.  It is similar to the randomized, incremental
-algorithms for convex hull and Delaunay triangulation.  We provide empirical
-evidence that the algorithm runs faster when the input contains non-extreme
+algorithms for convex hull and Delaunay triangulation. We provide empirical evidence that the algorithm runs faster when the input contains non-extreme
```

Comment:
Please restore this file to *exactly* the state it was in before you merge.  You have combined multiple lines into one line here.  

This file should not have been modified in any way, and should not be included in a merge.